homework/style.css

```diff
@@ -1,3 +1,151 @@
+body {
```

Comment:
The styling of the non-classes version can be improved. The classes version is better. The underline in the contributor list items should go.

![image](https://user-images.githubusercontent.com/2788771/60709219-36407d80-9f10-11e9-94e8-40a064b512a0.png)

![image](https://user-images.githubusercontent.com/2788771/60709246-448e9980-9f10-11e9-969c-0053cb3c0138.png)